src/gateway/delegate-routing.ts

```diff
@@ -31,11 +31,31 @@ const onlines: { [index: string]: RoutingData } = {};
 router.use(
   "/findpeer",
   mixGetPost,
+  rateLimit({
+    points: 100,
```

Comment:
Let's add configs to default.gateway.conf.json

```
delegates: {
  rateLimit: {
     findPeerEnabled; true,
     discoveryEnabled: true,
     findPeerLimit: 100,
     findPeerDuration: 10 mins,
     discoveryLimit: 100,
     discoveryDuration: 1 hour
   }
}

Reply:
The default values need to be adjusted.
Let's set 100 in 1 hour for discovery and disable findPeer limits. findPeer will be called for NOT_FOUND peers again and again and it could be an issue on deployers if we enable rate limit.